format/spec.md

```diff
@@ -44,6 +44,15 @@ The primary change in version 2 adds delete files to encode rows that are delete
 
 In addition to row-level deletes, version 2 makes some requirements stricter for writers. The full set of changes are listed in [Appendix E](#version-2).
 
+#### Version 3: Extended Types and Capabilities
+
+Version 3 of the Iceberg spec extends data types and existing metadata structures to add new capabilities, including:
+
+* Adding new data types, like nanosecond timestamps, variant, and unknown.
+* Adding new supported type promotion cases, integer and long to string
+* Adding default value support to struct fields
+* Adding multi-argument transforms for partitioning and sorting
+
```

Comment:
and specifies forwards compatibility for unknown transforms?

Reply:
If a reader doesn't recognize the transform, it will ignore it during query planning. For writing, an exception will be raised.

Reply:
I don't think this is worth including in the summary, but we could add that. To me, this is supposed to be high level and changes like that are a bit too in depth.